<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileViewer</class>
 <widget class="QMainWindow" name="ProfileViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="number">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>021 123 4567</string>
    </property>
   </widget>
   <widget class="QLabel" name="workHours">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>09:00 - 17:30  UTC</string>
    </property>
   </widget>
   <widget class="QLabel" name="JobTitle">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Senior Software Engineer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Greg</string>
    </property>
   </widget>
   <widget class="QLabel" name="PFP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>199</width>
      <height>199</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="email">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>almightyGreg@gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="location">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>390</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ffffff</string>
    </property>
    <property name="text">
     <string>Washington, DC</string>
    </property>
   </widget>
   <widget class="Line" name="navBackground">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#010511;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1301</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="NavButton_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="profileButton_2">
         <property name="styleSheet">
          <string notr="true">font-size: 24px;font-family: Inter;color: #32ACBE; font-weight:bold; border:none;</string>
         </property>
         <property name="text">
          <string>Profile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#05448A;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>15</number>
         </property>
         <property name="midLineWidth">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="NavButton_9">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="assignButton_2">
         <property name="styleSheet">
          <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
         </property>
         <property name="text">
          <string>Assigned</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_10">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#32ACBE;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>15</number>
         </property>
         <property name="midLineWidth">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="NavButton_10">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="managementButton_2">
         <property name="styleSheet">
          <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
         </property>
         <property name="text">
          <string>Management</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_11">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#32ACBE;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>15</number>
         </property>
         <property name="midLineWidth">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="NavButton_11">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="logoutButton_2">
         <property name="styleSheet">
          <string notr="true">font-size: 24px;font-family: Inter;color: rgb(255, 255, 255); font-weight:bold; border:none;</string>
         </property>
         <property name="text">
          <string>Logout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_12">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#CA0736;</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>15</number>
         </property>
         <property name="midLineWidth">
          <number>6</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Background">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>1321</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x0:1, y2:1, x2:0, y2:1, stop:0 rgba(1, 15, 17, 255), stop:1 rgba(4, 15, 101, 255))</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Line1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>611</width>
      <height>5</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>5</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#32ACBE; border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Line2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>611</width>
      <height>5</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>5</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#32ACBE; border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>number</zorder>
   <zorder>workHours</zorder>
   <zorder>JobTitle</zorder>
   <zorder>Name</zorder>
   <zorder>PFP</zorder>
   <zorder>email</zorder>
   <zorder>location</zorder>
   <zorder>navBackground</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>Line1</zorder>
   <zorder>Line2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
